lib/web/apiserver.go

```diff
@@ -322,6 +327,21 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	// Issue host credentials.
 	h.POST("/webapi/host/credentials", httplib.MakeHandler(h.hostCredentials))
 
+	h.GET("/webapi/resources/roles", h.WithAuth(h.getRolesHandle))
```

Comment:
`/webapi/resources/roles` ->  `/webapi/roles`

Lets remove the `/resources/` infix in all new endpoints

Reply:
revised